### ISSUE TITLE
uglify fails on the resulting files with Unexpected token name «get»

### ISSUE THREAD

**Issue by rosskevin** [rollup/rollup-plugin-babel#22]
Source files are here:

https://github.com/FezVrasta/bootstrap-material-design/tree/v4-dev

Uglify line failure on es6 file:

```
{ message: 'Unexpected token name «get», expected punc «(»',
  filename: 'bootstrap-material-design.es6.js',
  line: 32,
  col: 26,
  pos: 1092,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:1526:18)\n    at js_error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:1534:11)\n    at croak (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2025:9)\n    at token_error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2033:9)\n    at expect_token (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2046:9)\n    at expect (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2049:36)\n    at function_ (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2309:9)\n    at eval (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2160:24)\n    at eval (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2072:24)\n    at eval (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2819:23)' }
```

Offending line:

```
function babelHelpers_get get(object, property, receiver) {
```

Uglify line failure on umd file:

```
{ message: 'Unexpected token name «get», expected punc «(»',
  filename: 'bootstrap-material-design.umd.js',
  line: 38,
  col: 28,
  pos: 1358,
  stack: 'Error\n    at new JS_Parse_Error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:1526:18)\n    at js_error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:1534:11)\n    at croak (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2025:9)\n    at token_error (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2033:9)\n    at expect_token (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2046:9)\n    at expect (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2049:36)\n    at function_ (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2309:9)\n    at eval (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2160:24)\n    at eval (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2072:24)\n    at block_ (eval at <anonymous> (/Users/kross/acme/bootstrap-material-design/node_modules/uglify-js/tools/node.js:24:4), <anonymous>:2352:20)' }
```

Offending line: 

```
  function babelHelpers_get get(object, property, receiver) {
```

This project is newly setup based on the starter project, so perhaps I'm missing something...

**Comment by Rich-Harris** [comment]
I've just released 2.3.3 which should fix this bug – let me know if it doesn't!

**Comment by rosskevin** [comment]
Confirmed fixed
